modmesh/core.py

```diff
@@ -73,6 +74,9 @@
     'Bezier3dFp64',
     'WorldFp32',
     'WorldFp64',
+
+    # TODO: have a way to enable this according to `USE_PYTEST_HELPER_BINDING`
```

Comment:
didn't find out a good way to enable `testhelper` in python when `USE_PYTEST_HELPER_BINDING` is set on C++.
there are some complicated ways, such as doing file editing in CMake, but I think it isn't worth it.